pages/chat.tsx

```diff
@@ -31,7 +47,7 @@ export default function Chat() {
       </div>
 
       <div id="signup" style={{ textAlign: "center" }}>
-        <Container style={{ maxWidth: 880 }}>
+        <Container style={{ maxWidth: 880 }} onClick={() => onJoinClick()}>
```

Comment:
This is a really fascinating problem - how do we instrument an iframe we don't have control over!   

Good call keeping track of if the event has been sent already (they will potentially click a few times).  Personally, I'm a keyboard-first browser, and I'll tab into the form... Will some of the other events work here?  `onKeyDown` for instance?

Also, are there any APIs/etc available from Comsor for them to fire off the tracking event for us?

Reply:
Commsor does have an API... but it doesn't do what we want.

I'm going with the idea from standup to just track visits to the page versus actual interaction with the page and updating this goal for now.